### DISCUSSION TITLE
FluentNumberField text right

### DISCUSSION THREAD

**Discussion by acmesoft-arg** [microsoft/fluentui-blazor#1281] [Q&A] [answered]
Is strange a numeric form with the numbers left aligned, I didn't found any property to aling numbers, tried with class and style but none works for me, surely there was something I didn't understand to align the numbers to the right. Can someone help me?

**Comment by dvoituron** [comment, marked as answer]
You could use this style:

```html
<style>
    fluent-number-field::part(control) {
        text-align: right;
    }
</style>
```

**Reply by acmesoft-arg** [reply]
Excellent!, now looks good! Thank you, very much!.
